extra/3-magic-8-ball.js

```diff
@@ -58,8 +87,37 @@ function shakeBall() {
 
   This function should expect to be called with any value which was returned by the shakeBall function.
 */
+let answer = shakeBall();
+console.log(answer);
+
 function checkAnswer(answer) {
-  //Write your code in here
+  if (
```

Comment:
this loop with possible answers is pretty messed up, I tried it locally, but it makes mistakes. Just try to remove answers from the very positive section, it's going to give you "true"